src/app/landing/landing.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <div class="img_box">
+    <img class="bg_img" [src]="landingImgUrl" />
+    <div class="landing_page_info">
+      <h1 class="landing_page_title">Your <span>home</span> - your rules</h1>
+      <h2 class="landing_page_description">
+        Expand your options with <span>Generic housing loan</span>
+      </h2>
+      <!-- <button mat-stroked-button color="primary">Calculator</button> -->
+      <div class="landing_page_btn_box">
```

Comment:
For the future, there's a good standard called BEM (Block-Element-Modifier) for naming CSS classes - https://getbem.com/. 
In this case, the name of the class here would be `landing-page__btn-box`, for example.